js-quantities/index.d.ts

```diff
@@ -1,79 +1,73 @@
-// Type definitions for JS-quantities
+// Type definitions for JS-quantities 1.6
 // Project: http://gentooboontoo.github.io/js-quantities/
-// Definitions by: William Comartin <https://github.com/wcomartin>
+// Definitions by: William Rummler <https://github.com/wrummler>
```

Comment:
sorry for late respobse.
please keep original author name & append you name below.